tests/cli/test_main_clean.py

```diff
@@ -69,119 +68,124 @@ def has_pkg(name: str, contents: Iterable[str | Path]) -> bool:
 # conda clean --force-pkgs-dirs
 def test_clean_force_pkgs_dirs(
     clear_cache,
+    test_recipes_channel: Path,
     conda_cli: CondaCLIFixture,
     tmp_env: TmpEnvFixture,
+    tmp_pkgs_dir: Path,
 ):
-    pkg = "zlib"
+    pkg = "small-executable"
 
-    with make_temp_package_cache() as pkgs_dir:
-        # pkgs_dir is a directory
-        pkgs_dir = Path(pkgs_dir)
-        assert pkgs_dir.is_dir()
+    # pkgs_dir is a directory
+    assert tmp_pkgs_dir.is_dir()
 
-        with tmp_env(pkg):
-            stdout, _, _ = conda_cli("clean", "--force-pkgs-dirs", "--yes", "--json")
-            json.loads(stdout)  # assert valid json
+    with tmp_env(pkg):
+        stdout, _, _ = conda_cli("clean", "--force-pkgs-dirs", "--yes", "--json")
```

Comment:
can we assert this here just in case? It was the original issue:

```
assert context.pkgs_dirs == (tmp_pkgs_dir,)
```

Reply:
this assertion is in the new `tmp_pkgs_dir` fixture

Reply:
It's only part of it. It tests that it's the first one, but it doesn't assert that it's the only one:

```
assert context.pkgs_dirs[0] == pkgs_dir_str
```

In `test_main_clean` we need `context.pkgs_dirs` to ignore user config. Otherwise prepopulated caches might affect the outcome of the test (e.g. the user cache at ~/.conda/pkgs already has `zlib` (now `small-package`?) due to other tests or CI configs, so conda won't download it, so the new tmp pkgs_dir is not populated).

Reply:
opted to mock `context.pkgs_dirs` instead of monkeypatching to resolve this issue, see https://github.com/conda/conda/pull/13511